cmd/doc.go

```diff
@@ -51,30 +52,31 @@ var docCmd = &cobra.Command{
 	Hidden: true,
 	Short:  "Generates documentation for the tool in Markdown format",
 	Long:   "Generates documentation for the tool in Markdown format, and stores them in the designated location",
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		// verify the output location
 		f, err := os.Stat(docCmdInput.outputLocation)
 		if err != nil && os.IsNotExist(err) {
 			// create the output location if it does not exist yet
 			if err = os.MkdirAll(docCmdInput.outputLocation, os.ModePerm); err != nil {
-				fmt.Printf("Unable to create output location due to error: " + err.Error())
-				os.Exit(1)
+				fmt.Printf("doc: Unable to create output location due to error: " + err.Error())
+				return errors.New("doc: Unable to create output location due to error: " + err.Error())
```

Comment:
formattting of "doc:" is not consistent in some commands there is " : " and in some its ": ", let make them all the same.